.circleci/config.yml

```
@@ -42,16 +50,32 @@ jobs:
       - run:
           name: build book
           command: just build
+
   lint-links:
-    machine:
-      image: <<pipeline.parameters.base_image>>
+    executor: base
     steps:
       - checkout
       - run:
           name: Lint check
           command: just lint-links
       - notify-failures-on-develop:
           channel: C055R639XT9 #notify-link-check
+
+  publish-pages:
+    executor: builder
+    steps:
+      - checkout
+      - install-dependencies
+      - run:
+          name: Build
+          command: just build
+      - run:
+          name: Add CNAME file
+          command: echo "specs.optimism.io" > ./book/html/CNAME
+      - utils/get-github-access-token
+      - utils/github-pages-deploy:
+          src-pages-dir: ./book/html
```

```suggestion
          pages-branch: gh-pages-test
          src-pages-dir: ./book/html
```
To test you can try on a separate branch first (instead of gh-pages directly)

---

Done that all worked!